src/html.pre.js

```diff
@@ -11,6 +11,24 @@
  */
 const { getAbsoluteUrl, wrapContent, toClassName } = require('./utils.js');
 
+function readBlockConfig($block) {
+  if (!$block) return {};
```

Comment:
no one liners.... didn't lint complain ?

Reply:
Nope... maybe we need to adjust `@adobe/eslint-config-helix`? 😉